packages/lesswrong/components/titles/LocalgroupPageTitle.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { registerComponent } from '../../lib/vulcan-lib';
+import { useSingle } from '../../lib/crud/withSingle';
+import { useLocation } from '../../lib/routeUtil';
+import { Helmet } from 'react-helmet';
+
+const LocalgroupPageTitle = ({siteName}: {
+  siteName: string,
+}) => {
+  const { params: {groupId} } = useLocation();
+  
+  const { document: group, loading } = useSingle({
+    documentId: groupId,
+    collectionName: "Localgroups",
+    fragmentName: "localGroupsBase",
+    fetchPolicy: 'cache-only',
```

Comment:
I'm assuming this is copy-pasted, but that surprises me. I guess it works because the document is definitely going to be requested already. I guess I want to call attention to it as notable, but if it works where you copy-pasted from, and it works for this page, I'll trust it.